vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -511,8 +512,8 @@ def _get_all_objs(
                 obj_list[vimtype].append({
                     "mor_type": mor_type,
                     "mor": obj,
-                    "hostname": hostname,
-                    "tags": tags + instance_tags
+                    "hostname": ensure_bytes(hostname) if hostname else hostname,
```

Comment:
Could we rather change the `ensure_byte` to return `None` if the given param is `None`?